.travis.yml

```diff
@@ -12,3 +12,4 @@ node_js:
   - "0.10"
   - "0.12"
   - 'iojs'
```

Comment:
io.js will most likely receive a minor update or two but no major ones so it will die soon anyway.

You may keep it for now but in a month or two no one will use it. :) Few devs switched from Node anyway and those who did will surely switch to Node 4 quickly.